packages/fxa-auth-server/lib/senders/emails/templates/postRemoveRecoveryPhone/en.ftl

```diff
@@ -0,0 +1,4 @@
+postRemoveRecoveryPhone-subject = Backup recovery phone removed
+postRemoveRecoveryPhone-title = Backup recovery phone removed
+postRemoveRecoveryPhone-description = Your backup recovery phone has been removed from your two-step authentication settings. You still have your backup recovery codes available for use.
```

Comment:
Looks like we missed a terminology update in Figma. This should be "backup authentication codes" - I've updated Figma to have the correct term.

```suggestion
postRemoveRecoveryPhone-description = Your backup recovery phone has been removed from your two-step authentication settings. You still have your backup authentication codes available for use.
```